docs/fr/README.md

```diff
@@ -0,0 +1,32 @@
+# Dictionnaire Français Pour Votre Liseuse
+
+**eBook Reader Dictionaries** est un projet collaboratif utilisant l’immense base de connaissances du [Wiktionnaire](https://fr.wiktionary.org/) pour créer un dictionnaire complet, mis à jour quotidiennement.
+
+## Fonctionalités Intéressantes
+
+- [Kobo] Lors de la sélection d’un mot au pluriel, sa forme au singulier sera affichée.
```

Comment:
Should we keep `[Kobo]` for the 3 lines?

Reply:
Yes, it's true after all.